### ISSUE TITLE
Animating the color of the SVG is not possible. 

### ISSUE THREAD

**Issue by yasinarik** [peiffer-innovations/websafe_svg#4]
Hi! Thanks for building this package. It works pretty well for me apart from animating the color.

I already made a custom SVG renderer which changes color by state changes. 

However, my custom version lacks the ability to animate the color over time. **websafe_svg** lacks too.

I believe that the source of the problem is `platformRegistery`.

Here is an example:

```dart
import 'package:flutter/material.dart';
import 'package:websafe_svg/websafe_svg.dart';

class AnimatedSvgExample extends StatefulWidget {
  @override
  _AnimatedSvgExampleState createState() => _AnimatedSvgExampleState();
}

class _AnimatedSvgExampleState extends State<AnimatedSvgExample> with SingleTickerProviderStateMixin {
/* ----------------------------- state variables ---------------------------- */
  Color color;
  AnimationController controller;
  Animation<Color> colorAnimation;

/* ----------------------------------- end ---------------------------------- */

/* ------------------------------- init state ------------------------------- */

  @override
  void initState() {
    super.initState();

    controller = AnimationController(
      duration: const Duration(milliseconds: 1000),
      vsync: this,
    );

    colorAnimation = ColorTween(begin: Colors.green, end: Colors.blue).animate(controller);

    controller.repeat(reverse: true, period: Duration(milliseconds: 1000));
  }

/* ----------------------------------- end ---------------------------------- */

/* ---------------------------- main build method --------------------------- */

  @override
  Widget build(BuildContext context) {
    return AnimatedBuilder(
      animation: colorAnimation,
      builder: (context, w) {
        print(colorAnimation.value.value.toRadixString(16).substring(2)); // Successfully prints. Color changes.

        return Container(
          margin: EdgeInsets.all(124),
          color: colorAnimation.value, // Color of the container itself changes
          child: WebsafeSvg.asset(
            "logo.svg",
            color: colorAnimation.value, // Unfortunately, it doesn't rebuild. It sticks with the initial color which is green.
          ),
        );
      },
    );
  }

/* ----------------------------------- end ---------------------------------- */

}

```

**Comment by jpeiffer** [comment]
Thanks for the detailed example.  Investigating now.

**Comment by jpeiffer** [comment]
This is fixed with version `1.10.0`, however, the fix also depends on a Flutter fix which is currently in the `dev` channel and not yet available in `beta`.  So you will need to update Flutter to `1.19.0-1.0.pre` or higher.
